cargo-shuttle/tests/integration/run.rs

```diff
@@ -85,11 +88,11 @@ async fn rocket_secrets() {
 // This example uses a shared Postgres. Thus local runs should create a docker container for it.
 #[tokio::test(flavor = "multi_thread")]
 async fn rocket_postgres() {
-    let port = cargo_shuttle_run("../examples/rocket/postgres").await;
+    let url = cargo_shuttle_run("../examples/rocket/secrets", false).await;
```

Comment:
You probably did a find & replace for these, it looks like you forgot to change the paths back to the correct example (for this and the following tests).

Reply:
Oh whoops 🤦 thanks for catching that, I'll send a fix in quickly

Reply:
No worries, been there :smile: Have a good weekend!

Reply:
Thanks, you too!